cores/esp8266/core_esp8266_postmortem.c

```diff
@@ -37,6 +37,11 @@ static void uart1_write_char_d(char c);
 static void print_stack(uint32_t start, uint32_t end);
 //static void print_pcs(uint32_t start, uint32_t end);
 
+extern void __custom_crash_callback( struct rst_info * rst_info, uint32_t stack, uint32_t stack_end ) {
+}
+
+extern void custom_crash_callback( struct rst_info * rst_info, uint32_t stack, uint32_t stack_end ) __attribute__ ((weak, alias("__custom_crash_callback")));
```

Comment:
I've aped the style [used elsewhere](https://github.com/esp8266/Arduino/blob/master/cores/esp8266/core_esp8266_wiring_digital.c#L177) [in this project](https://github.com/esp8266/Arduino/blob/master/cores/esp8266/core_esp8266_main.cpp#L96), though I must admit I don't know the reason for the alias (my guess is that it makes for less confusion when looking at stack dumps).